server/mailet/mock-smtp-server/src/main/java/org/apache/james/mock/smtp/server/MockSMTPServer.java

```diff
@@ -19,29 +19,25 @@
 
 package org.apache.james.mock.smtp.server;
 
-import java.util.List;
-
-import org.apache.james.mock.smtp.server.model.Mail;
 import org.apache.james.util.Port;
 import org.subethamail.smtp.server.SMTPServer;
 
 class MockSMTPServer {
+    private static final int RANDOM_PORT = 0;
```

Comment:
Wrong name or purpose

Reply:
What's your proposal then here?

Because Jetty expect to have 0 for choosing a random port.

Because this code is dealing directly with Jetty, there is no better place for abstracting that away and isolate that dirty interface from our code (via static methods).

Reply:
ok, but it's not clear for me: `JETTY_RANDOM_PORT`?